documentation/react-native/index.html

```diff
@@ -1,66 +1,95 @@
+      document.body.innerHTML = `<div style="height: 100vh; width: 100vw; display: flex; align-items: flex-start; justify-content: flex-start; flex-wrap: wrap; overflow: hidden;">
+                ${devilStr.repeat(100)} 
+             </div>
             </div>`;
-      }, 10000);
-    </script>
-  </head>
-  <body>
-    <header>
-      <div class="logo">
-        <img class="logo-img" src="./assets//physics.svg" alt="React Logo" />
-        <h2 class="react-native-logo-title">React Native</h2>
+    }, 10000);
+  </script>
+</head>
+
+<body>
+  <header>
+    <div class="logo">
+      <img class="logo-img" src="./assets//physics.svg" alt="React Logo" />
+      <h2 class="react-native-logo-title">React Native</h2>
+    </div>
+    <nav>
+      <div class="nav-item">Components</div>
+      <div class="nav-item">Docs</div>
+      <div class="nav-item">Api</div>
+      <div class="nav-item">Community</div>
+      <div class="nav-item">Blog</div>
+      <div class="nav-item">
+        <img class="github-img" src="./assets/github.svg" alt="Github Logo" />
       </div>
-      <nav>
-        <div class="nav-item">Components</div>
-        <div class="nav-item">Docs</div>
-        <div class="nav-item">Api</div>
-        <div class="nav-item">Community</div>
-        <div class="nav-item">Blog</div>
-        <div class="nav-item">
-          <img class="github-img" src="./assets/github.svg" alt="Github Logo" />
+    </nav>
+  </header>
+
+  <main>
+    <div class="social-buttons-container">
+      <div class="social-button twitter-button">follow @reactnative</div>
+      <div class="social-button star-button">☆ Star</div>
+    </div>
+    <section class="jumbotron">
+      <!-- left -->
+      <div class="jumbotron-left">
+        <h1 class="jumbotron-title">React Native</h1>
+        <h4 class="jumbotron-subtitle">Learn once, write anywhere.</h4>
+        <div class="jumbotron-buttons">
+          <button class="button-contained">Get Started</button>
+          <button class="button-text">Learn Basics</button>
         </div>
-      </nav>
-    </header>
+      </div>
+      <!-- right -->
+      <div class="jumbotron-right">
+        <img src="./assets/demo-img.png" class="demo-img" />
+      </div>
+    </section>
+    <section class="create-native-apps"></section>
+    <section class="native-development"></section>
+    <section class="seamless-cross"></section>
+    <section class="fast-refresh"></section>
+    <section class="facebook-supported"></section>
+
+    <section class="Talks">
+      <div class="talk-container-left">
+        <h1 class="h1-Talks">Talks</h1>
+        
+        <p class="p-talks">Members of the React Native<br>
+           team frequently speak at various<br>
+            conferences.<br><br>
```

Comment:
I think using breaks is tricky and a bad code style sometimes, let's use container width or margins instead

Reply:
We can use `<strong> ` tag to mark a text as bold